pkg/daemons/config/types.go

```diff
@@ -149,7 +148,6 @@ type CriticalControlArgs struct {
 	DisableNPC            bool         `cli:"disable-network-policy"`
 	DisableServiceLB      bool         `cli:"disable-service-lb"`
 	EncryptSecrets        bool         `cli:"secrets-encryption"`
-	MultiClusterCIDR      bool         `cli:"multi-cluster-cidr"`
```

Comment:
We should validate that this doesn't break compatibility between servers that have this field, and servers that don't. I think it should be OK, and only error if it was set to true - but still would be good to check.

Reply:
I see, I will add this note to the issue to be validated, I will also add a note in the release note for this, since its a breaking change from 1.28 to 1.29